src/routes/error.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [{
+  method: ['GET', 'POST'],
```

Comment:
Guessing its just a copy+paste thing, but just checking if you think there is a need to handle POST requests on the error page?